app/api/threads/list_threads.feature

```diff
@@ -0,0 +1,172 @@
+      | Baptiste     | BaptisteCanViewInfo                   |               | info                     |           |
+      | Baptiste     | BaptisteCanViewContent1               |               | content                  |           |
+      | Baptiste     | BaptisteCanViewContent2               |               | content                  |           |
+      | Baptiste     | BaptisteCanViewContentWithDescendants |               | content_with_descendants |           |
+      | EtienneKlein | EtienneKleinHasValidated1             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasValidated2             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasValidated3             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasValidated4             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasValidated5             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasValidated6             | 1             |                          |           |
+      | EtienneKlein | EtienneKleinHasNotValidated           |               |                          | answer    |
+      | EtienneKlein | EtienneKleinCanWatchAnswer1           |               |                          | answer    |
+      | EtienneKlein | EtienneKleinCanWatchAnswer2           |               |                          | answer    |
+      | EtienneKlein | EtienneKleinCanWatchAnswer3           |               |                          | answer    |
+      | EtienneKlein | EtienneKleinCanWatchAnswer4           |               |                          | answer    |
+    Given there are the following threads:
+      | participant         | item                                  | helper_group         | status                  | latest_update_at     | comment                                                                                                                |
+      | PresidentConsortium |                                       |                      |                         |                      |                                                                                                                        |
+      | PresidentUniversity |                                       |                      |                         |                      |                                                                                                                        |
+      | Mary                |                                       |                      |                         |                      |                                                                                                                        |
```

Comment:
Threads have participant and item as key so I don't know how to interpret this. I should not have to try to guess what it means.

Reply:
In this case it means that the item can be whatever item because its parameters don't matter.

The alternative would be to do:
```
participant	item
ConsortiumPresident	Item1
UniversityPresident	Item2
```

But since we don't need to define anything more on those items, that would be the only place where they appear. Would that be clear enough?

Reply:
> In this case it means that the item can be whatever item because its parameters don't matter.

I don't understand what you mean

Reply:
What I mean is that we don't need to define anything special about this item, the only thing is that it must exist so that the thread can be created, to fulfill database constraints.